admin/spec/service/group.service.spec.js

```diff
@@ -57,6 +70,18 @@ describe('group.service', () => {
       expect(pupils).toEqual(pupilsMock)
       done()
     })
+
+    it('should return an error if schoolId is missing', async (done) => {
+      const schoolId = null
+      const groupIdToExclude = 1
+      try {
+        await groupService.getPupils(schoolId, groupIdToExclude)
+        done()
```

Comment:
this will silently fail if an error is not thrown.  you should do something like this `fail('error not thrown')`